rpcs3/Emu/Cell/lv2/sys_spu.cpp

```diff
@@ -18,9 +18,119 @@ namespace vm { using namespace ps3; }
 
 logs::channel sys_spu("sys_spu");
 
+void sys_spu_image::load_from_memory(u32 data_addr, u32 type)
```

Comment:
I cannot review this copypasta. It should be an additional argument within the same function, if its behaviour needs to change.